hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/TestHoodieClientOnCopyOnWriteStorage.java

```diff
@@ -437,7 +456,10 @@ private void testUpsertsInternal(HoodieWriteConfig config,
    */
   @Test
   public void testDeletes() throws Exception {
-    SparkRDDWriteClient client = getHoodieWriteClient(getConfig(), false);
+    // Set cleaner to LAZY so no inflights are cleaned
```

Comment:
why can't we override this once for all tests. if we run everything with `LAZY` then we can have better confidence that no uncommitted data will ever be exposed